example/android/app/build.gradle

```diff
@@ -52,6 +49,10 @@ android {
             signingConfig signingConfigs.debug
         }
     }
+    namespace 'com.xraph.plugin.flutter_unity_widget_example'
```

Comment:
In the Flutter repo they make this conditional, To also handle old gradle versions that don't include namespace. https://github.com/flutter/flutter/issues/125621#issuecomment-1525995461

```
 if (project.android.hasProperty("namespace")) {
        namespace 'your.namespace.here'
 }
```
I did the same in https://github.com/juicycleff/flutter-unity-view-widget/pull/807, but that hasn't been reviewed or merged yet.

Reply:
Sounds good! Hadn't seen your AGP 8 pull request. I went for a more conservative upgrade to 7.4.2. If we can get 8 to work that would be better. Sorry, I'm new to PRs, maybe I should have split this into multiple smaller PRs

Reply:
Flutter [reccomends](https://docs.flutter.dev/release/breaking-changes/android-java-gradle-migration-guide) using versions 7.3-7.6.1, so 7.4.2 is fine.

I mention 8 in the pull request because in 8.x the missing namespace results in errors instead of the warnings/reccomendations for 7.x .